code/system/application/sensation/src/geolocation/geolocation.cpp

```diff
@@ -157,25 +170,25 @@ odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode Geolocation::body()
         << timestamp << "\t hasData=" << hasData << "\tx=" << state.x() << ", y=" <<
         state.y() << ", theta=" << state.theta() << std::endl;
 
-      // Build the proper data to send
+      // Build the proper GPS coordinates to send
       opendlv::data::environment::Point3 currentStateEstimation (state.x(), state.y(), currentCartesianLocation.getZ());
-      opendlv::data::environment::WGS84Coordinate currentCartesianLocationEstimation = gpsReference.transform(currentStateEstimation);
+      opendlv::data::environment::WGS84Coordinate currentWGS84CoordinateEstimation = gpsReference.transform(currentStateEstimation);
       double heading = state.theta();
 
       std::cout   << getName() << "\t" << "timestamp="
-        << timestamp << "\t hasData=" << hasData << "\tlat=" << currentCartesianLocationEstimation.getLatitude() << ", long=" <<
-        currentCartesianLocationEstimation.getLongitude() << ", theta=" << state.theta() << std::endl;
+        << timestamp << "\t hasData=" << hasData << "\tlat=" << currentWGS84CoordinateEstimation.getLatitude() << ", long=" <<
+        currentWGS84CoordinateEstimation.getLongitude() << ", theta=" << state.theta() << std::endl;
 
       // Send the message
-      opendlv::sensation::Geolocation geoLocationEstimation(currentCartesianLocationEstimation.getLatitude(),
-                                                            currentCartesianLocationEstimation.getLongitude(),
+      opendlv::sensation::Geolocation geoLocationEstimation(currentWGS84CoordinateEstimation.getLatitude(),
+                                                            currentWGS84CoordinateEstimation.getLongitude(),
                                                             gpsReading.getAltitude(),
                                                             heading);
       odcore::data::Container msg(geoLocationEstimation);
       getConference().send(msg);
 
 
-    }
+    } else cout << " NO DATA " << endl;
```

Comment:
Please always use brackets, and use complete namespace details:
} else {
  std::cout << "No data" << std::endl;
}